Podfile

```diff
@@ -1,22 +1,30 @@
 workspace 'VimeoUpload'
+xcodeproj 'Framework/VimeoUpload/VimeoUpload.xcodeproj'
 xcodeproj 'Examples/VimeoUpload-iOS/VimeoUpload-iOS.xcodeproj'
 xcodeproj 'Examples/VimeoUpload-iOS-OldUpload/VimeoUpload-iOS-OldUpload.xcodeproj'
 xcodeproj 'Examples/VimeoUpload-OSX/VimeoUpload-OSX.xcodeproj'
 
 #install! 'cocoapods', :deterministic_uuids => false # This suppresses the duplicate UUID warnings, introduced in Cocoapods 1.0
 # https://github.com/CocoaPods/CocoaPods/issues/4370#issuecomment-183205691
 
+target 'VimeoUpload' do
+    platform :ios, '8.0'
+    use_frameworks!
+    pod 'VimeoNetworking', :git => 'git@github.com:vimeo/VimeoNetworking.git', :branch => 'master'
+    xcodeproj 'Framework/VimeoUpload/VimeoUpload.xcodeproj'
+end
+
 target :'VimeoUpload-iOS' do
     platform :ios, '8.0'
     use_frameworks!
-    pod 'VIMNetworking/Model', '6.0.4'
+    pod 'VimeoNetworking', :git => 'git@github.com:vimeo/VimeoNetworking.git', :branch => 'master'
```

Comment:
should we define this in a shared section since it's included in all the targets?